examples/language-modeling/run_lora_clm.py

```diff
@@ -612,13 +616,13 @@ def concatenate_data(dataset, max_seq_length):
         if data_args.dataset_name == "tatsu-lab/alpaca" or data_args.sql_prompt:
             tokenized_datasets_ = tokenized_datasets["train"].remove_columns(["prompt_sources", "prompt_targets"])
             if training_args.do_eval:
-                tokenized_datasets_eval_ = tokenized_datasets["validation"].remove_columns(
+                tokenized_datasets_eval_ = tokenized_datasets[data_args.eval_split].remove_columns(
                     ["prompt_sources", "prompt_targets"]
                 )
         elif data_args.dataset_name == "timdettmers/openassistant-guanaco":
             tokenized_datasets_ = tokenized_datasets["train"].remove_columns(["input", "output"])
             if training_args.do_eval:
-                tokenized_datasets_eval_ = tokenized_datasets["test"].remove_columns(["input", "output"])
+                tokenized_datasets_eval_ = tokenized_datasets[data_args.eval_split].remove_columns(["input", "output"])
```

Comment:
should we add --eval_split to "test" to falcon to get the same dataset as before?

Reply:
Since you managed to reach the same perf as before using a different validation split percentage, I think we can just close this PR to not overcomplicate things. WDYT?

Reply:
sure. let's close it.